src/Firmware/Weighbridge/snake.h

```diff
@@ -144,15 +144,15 @@ namespace SNAKE_MODE {
       display.printf("Score: %d", tail->size);
       display.display();
       
-      StaticJsonBuffer<BUFFER_SIZE> jsonBuffer;
+      StaticJsonBuffer<JSON_OBJECT_SIZE(4)> jsonBuffer;
       JsonObject& json = jsonBuffer.createObject();
 
       json["time_played"] = millis() - gameTime;
-      json["score"] = tail.size;
+      json["score"] = tail->size;
 
       char message[json.measureLength() + 1];
       json.printTo(message, sizeof(message));
-      publish(MQTT_SNAKE_STATE, message)
+      publish(MQTT_SNAKE_STATE, message);
```

Comment:
`MQTT_SNAKE_STATE` is not defined but `MQTT_SNAKE_CHANNEL`. I suggest for convenience to define it this way in `sample-config.h`
`#define MQTT_CHANNEL_SNAKE            "/weighbridge/api/1/snake/"`